ml-agents/mlagents/trainers/sac/trainer.py

```diff
@@ -257,7 +283,7 @@ def update_sac_policy(self) -> None:
         for stat, stat_list in batch_update_stats.items():
             self.stats_reporter.add_stat(stat, np.mean(stat_list))
 
-        bc_module = self.sac_policy.bc_module
+        bc_module = self.policy.bc_module
```

Comment:
I'm surprised this works since self.policy should be a TFPolicy not a SACPolicy

Reply:
Is it because I removed self.policy from trainer?  self.policy now just exist at the level of ppo/sac